kex

```diff
@@ -43,7 +45,10 @@ EOF
 }
 
 po_list() {
-  kubectl get pods -o=jsonpath='{range .items[*].metadata.name}{@}{"\n"}{end}'
+  local NS_PARAM="--namespace=${NAMESPACE:-}"
```

Comment:
Thanks for the PR!  In testing, this now creates an order dependency.  If I place the `-n` switch after `-l` or `-p`, the current-context namespace remains in play.  Is it possible to be order agnostic, or do we need to change the instructions to clarify that, when namespace is called, it must be the first operator in order to be successful?

Here is my test.  The first command shows that order has failed, although the API responds without an error.  The second shows the correct `namespace`-specific pod list:
```
$ ./kex -l -n mps
     1	mq-rabbitmq-905285719-jj01p
     2	nginx-deployment-2122188915-7xrll
     3	nginx-deployment-2122188915-8rfq5
     4	nginx-deployment-2122188915-dgpzh
     5	sqe-wiremock-8xpl6
     6	tiller-deploy-553736225-s18vh
     7	warped-clam-jenkins-501160700-2cr2h

josh206534426 at 5TSDPIMTEMP02LM in ~/src/github.com/farmotive/kex on master [!]
$ ./kex -n mps -l
     1	db-1101631534-mp3s3
     2	mps-web-php-3056204531-0hbmd
     3	sysdig-agent-7g4pb
     4	sysdig-agent-x3pkm
     5	sysdig-agent-xzl2k
```